src/api.js

```diff
@@ -133,15 +153,15 @@ export default class BounceClient {
         const attrs = {
```

Comment:
in line 152 above, it might be better to have an object of URLs in the future since there could be more links